device/samples/DeviceClientAmqpSample/Program.cs

```diff
@@ -15,6 +15,10 @@ class Program
         private const string DeviceConnectionString = "<replace>";
 
         private static int MESSAGE_COUNT = 5;
+        private static String deviceId = "MyCSharpDevice";
```

Comment:
>deviceId [](start = 30, length = 8)

DeviceId is provided in the connection string, so no need for this variable. Here and in the other samples.